tasks/instagrampoll.php

```diff
@@ -60,134 +60,137 @@ public function cleanup()
 
 	public function subscriptions()
 	{
-		$account = \Propeller\Instagram\Model_Account::query()
+		try 
+		{
```

Comment:
Could we keep all braces for anything other than functions and classes on the same line please?